core/src/main/scala/org/apache/pekko/persistence/jdbc/state/DurableStateQueries.scala

```diff
@@ -46,6 +47,9 @@ import slick.jdbc.{ H2Profile, JdbcProfile, OracleProfile, PostgresProfile, SQLS
   private[jdbc] def selectFromDbByPersistenceId(persistenceId: Rep[String]) =
     durableStateTable.filter(_.persistenceId === persistenceId)
 
+  private[jdbc] def selectFromDbByPersistenceId(persistenceId: String) =
+    durableStateTable.filter(_.persistenceId === persistenceId)
```

Comment:
I think this is duplicate, if you want to add a method that is used from outside, you may want to implement it like this:(i didn't verify it)

```
def selectFromDbByPersistenceId(persistenceId: String) =
    db.run(durableStateTable.filter(_.persistenceId === persistenceId).result)
```

Reply:
removed